src/Badge.svelte

```diff
@@ -12,20 +12,21 @@
     display: flex;
   }
   .foreground {
-    color: var(--color-foreground-faded);
-    background: var(--color-foreground-background);
+    color: var(--color-foreground-6);
+    background: var(--color-foreground-2);
   }
   .primary {
     color: var(--color-primary);
-    background: var(--color-primary-background);
+    background: linear-gradient(var(--color-primary-2),var(--color-primary-2)),
+    linear-gradient(var(--color-background), var(--color-background));
```

Comment:
@brandonhaslegs wonder if there's another way to get this work without having to mix the colors? I assume you already tried to fiddle with the actual contrasts of the colors to get it right?

Reply:
I suppose we could flatten all the colors… The only reason I'm doing this here is because in this case, two colors are stacked, and since they're transparencies, they mix and look muddy when stacked. Normally I wouldn't stack colors, but that's just how the design was in this case. Keeping the color levels as transparencies is easy for me, but I guess we could just flatten them… I wish you could stack colors without using a css gradient… it certainly looks dumb…

Reply:
Hm, good point. Yeah, I agree, ideally we shouldn't use transparencies, so we can layer stuff better. I previously tried to convert everything to HSL, but there wasn't an obvious pattern, because we'd have to mix in some of the background into each of the colors. I'll check if there's another way to programmatically achieve this, so that we don't have to flatten them every time by hand.

Reply:
I can flatten them by hand, but that seems like an overreaction to this one little "problem" of using a css gradient to layer color styles.

Reply:
Yeah, you're probably right. Let's punt on this for now.